mesonbuild/mesonlib.py

```diff
@@ -270,8 +270,10 @@ def from_absolute_file(fname: str) -> 'File':
     def rel_to_builddir(self, build_to_src: str) -> str:
         if self.is_built:
             return self.relative_name()
+            #NO! return os.path.realpath(self.relative_name())
         else:
             return os.path.join(build_to_src, self.subdir, self.fname)
+            #NO! return os.path.realpath(os.path.join(build_to_src, self.subdir, self.fname))
```

Comment:
What are these comments for? I don't understand what they try to say?

Reply:
My intension was to use realpath() , but there is still a ninja bug.
see https://github.com/ninja-build/ninja/issues/1251#issuecomment-599541385

Reply:
done